### PR TITLE
add troubleshooting section to bulk upload

### PR DESCRIPTION
**Pull request by mrjones-plip**
Covers troubleshooting for:
* Access denied
* Wrong type, this is not a person

### CODE REVIEW

**Comment by mrjones-plip** [comment]
cc @tatilepizs

File: content/en/apps/guides/data/users-bulk-load.md
Comment (m5r) [inline content/en/apps/guides/data/users-bulk-load.md:139]:
```suggestion
As of CHT 3.7.0, you're [allowed to declare different contact types]({{< relref "apps/reference/app-settings/hierarchy#app_settingsjson-contact_types" >}}) in your `app_settings.json`. If you have populated the `.contact_types[]` property in your JSON, you will need to update the automatic value of the `contact.contact_type` column.  The default value is:
```

I had never seen the word `stanza` before but it's pretty satisfying to say out loud :joy:  
Since the audience of these docs are usually not native English speakers, I think it would be safer to fall back to more common words

**Review by m5r — APPROVED** [review]
Great addition, thanks for your work!  
I left one suggestion inline but I'm already approving this PR so you don't have to wait for my approval when I come back on monday

File: content/en/apps/guides/data/users-bulk-load.md
Comment (mrjones-plip) [reply content/en/apps/guides/data/users-bulk-load.md:139]: Ah - it is a great word!  I'll go with the more common suggestion - thanks!

**Pull request merged**
